### PR TITLE
test: check extreme DATE/DATETIME values can be loaded from pandas DataFrame

### PR DESCRIPTION
**Pull request by tswast**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/python-bigquery/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Towards #1076 🦕
(edit: moved to https://github.com/googleapis/python-db-dtypes-pandas/issues/45 )

### CODE REVIEW

File: tests/system/test_pandas.py [hunk 1]
```diff
@@ -283,15 +283,6 @@ def test_load_table_from_dataframe_w_nulls(bigquery_client, dataset_id):
...
 
-    table_schema = scalars_schema + (
-        # TODO: Array columns can't be read due to NULLABLE versus REPEATED
-        #       mode mismatch. See:
-        #       https://issuetracker.google.com/133415569#comment3
-        # bigquery.SchemaField("array_col", "INTEGER", mode="REPEATED"),
-        # TODO: Support writing StructArrays to Parquet. See:
-        #       https://jira.apache.org/jira/browse/ARROW-2587
-        # bigquery.SchemaField("struct_col", "RECORD", fields=scalars_schema),
-    )
```
Comment (tseaver) [inline tests/system/test_pandas.py:294]: Are we just bailing on those two `TODO` entries?

**Review by tseaver — APPROVED** [review]

**Comment by plamut** [comment]
> Towards #1076 

Where's that issue, is this a typo?
(there's no cross-reference)

File: tests/system/test_pandas.py (hunk 1 above)
Comment (plamut) [reply tests/system/test_pandas.py:294]: +1, is this now off-plan?

**Review by plamut — APPROVED** [review]
Looks good, assuming that simply removing those TODOs is indeed intended.

File: tests/system/test_pandas.py (hunk 1 above)
Comment (tswast) [reply tests/system/test_pandas.py:294]: These tests are actually already covered in the v3 branch. https://github.com/googleapis/python-bigquery/blob/3b3ebff56004e061f58ecdd06fc9020423b890e2/tests/system/test_pandas.py#L294

**Pull request merged**